clients/lsp-futhark.el

```diff
@@ -0,0 +1,38 @@
+;;; lsp-futhark.el --- lsp-mode futhark integration    -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2024 lsp-mode maintainers
+
+;; Keywords: languages
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Client for the futhark language server.
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(add-to-list 'lsp-language-id-configuration '(futhark-mode . "futhark"))
```

Comment:
I don't think you need this anymore since you have already edit the `lsp-mode.el` file.

Reply:
Yeah, that makes sense.

Reply:
I added the require lsp-mode as that's what was in the config I was looking at (nix)